sk0-back/src/users/services/password.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@nestjs/common';
+import * as bcrypt from 'bcrypt';
+
+@Injectable()
+export class PasswordService {
+  async hashPassword(password: string): Promise<string> {
+    const saltRounds = 10;
+    return bcrypt.hash(password, saltRounds);
+  }
+
+  async comparePasswords(plainTextPassword: string, hashedPassword: string): Promise<boolean> {
+    return bcrypt.compare(plainTextPassword, hashedPassword);
+  }
```

Comment:
I am not convinced that we need to add the suffix "password" to a password service.
It is leading to: `passwordService.hasPassword()`. If this service does nothing else, the password should be in scope and - in my opinion - should not be in the name of the methods. What do you think about this?

Reply:
totally good point, I replace it